code/game/objects/items/storage/pill_packets.dm

```diff
@@ -49,6 +49,14 @@
 	storage_slots = 8
 	max_storage_space = 8
 
+/obj/item/storage/pill_bottle/packet/isotonic
+	name = "Isotonic pill packet"
```

Comment:
```suggestion
	name = "\improper Isotonic pill packet"
```

Reply:
Why `/improper`, none of the other pill packet names have it?

Reply:
It's for autoformating in sentences. Anything that isn't a proper noun and starts with a capital should have it.